install.sh

```diff
@@ -49,16 +52,68 @@ else
 	fi
 fi
 
+
+allBuildArguments=("$@")
+for ((index=0; index < ${#allBuildArguments[@]}; index++)); do
+	if [ ${allBuildArguments[index]} = '-r' ]; then
+		reposArg=${allBuildArguments[index+1]}
+		if [ "$reposArg" = "" ]; then
```

Comment:
why the difference between line 60 and line 77?

Reply:
Because line 77 use 2 conditions using or either I need to use:
[[ x$reposArg == 'x' || x$reposArg == "" ]]
or 
[ x$reposArg == 'x' ] || [ x$reposArg == "" ]
 
Anyway, I will changed it to use the later to be more clear.

Reply:
Will remove the check for empty as there is already check earlier that will fail the script if user do not specify the value with -r option.